tests/visualization_tests/test_parallel_coordinate.py

```diff
@@ -129,6 +129,58 @@ def test_plot_parallel_coordinate_categorical_params() -> None:
     assert figure.data[0]["dimensions"][2]["ticktext"] == ("net", "una")
 
 
+def test_plot_parallel_coordinate_categorical_numeric_params() -> None:
+    # Test with categorical params that can be interpreted as numeric params.
+    study_categorical_params = create_study()
+    study_categorical_params.add_trial(
+        create_trial(
+            value=0.0,
+            params={"category_a": 2, "category_b": 20},
+            distributions={
+                "category_a": CategoricalDistribution((1, 2)),
+                "category_b": CategoricalDistribution((10, 20, 30)),
+            },
+        )
+    )
+    study_categorical_params.add_trial(
+        create_trial(
+            value=1.0,
+            params={"category_a": 1, "category_b": 30},
+            distributions={
+                "category_a": CategoricalDistribution((1, 2)),
+                "category_b": CategoricalDistribution((10, 20, 30)),
+            },
+        )
+    )
+    study_categorical_params.add_trial(
+        create_trial(
+            value=2.0,
+            params={"category_a": 2, "category_b": 10},
+            distributions={
+                "category_a": CategoricalDistribution((1, 2)),
+                "category_b": CategoricalDistribution((10, 20, 30)),
+            },
+        )
+    )
+
+    # Trials are sorted by using param_a and param_b.
```

Comment:
Could you add the expected order of trial in concrete? 
```suggestion
    # Trials are sorted by using param_a and param_b, i.e., trial#1, trial#2, and trial#0.
```